tests/test.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+import pandas as pd
+from pandas import read_csv
+from pathlib import Path
+
+from src.run_cocci import run_cocci
+from src.option_select import select
+
+### Modify this later
+patch_to_input_mapping = {
+    "assignment_as_value.cocci": ["assignment_as_value.c"],
+    "change_of_literal_encoding.cocci": ["change_of_literal_encoding.c"],
+    "comma_operator.cocci": ["comma_operator.c"],
+    "conditional_operator.cocci": ["conditional_operator.c"],
+    "implicit_predicate.cocci": ["implicit_predicate.c"],
+    "logic_as_controlflow.cocci": ["logic_as_controlflow.c"],
+    # "macro_operator_precedence.cocci": [".c"],
+    "omitted_curly_braces.cocci": ["omitted_curly_braces.c"],
+    # "operator_precedence.cocci": [".c"],
+    "post_incdec.cocci": ["post_incdec.c"],
+    "pre_incdec.cocci": ["pre_incdec.c"],
+    "repurposed_variable.cocci": ["repurposed_variable.c"],
+    "reversed_subscripts.cocci": ["reversed_subscripts.c"],
+    "type_conversion.cocci": ["type_conversion.c"],
+}
+
+### Modify this later
+input_to_expected_mapping = {
+    "assignment_as_value.c": "assignment_as_value.csv",
+    "change_of_literal_encoding.c": "change_of_literal_encoding.csv",
+    "comma_operator.c": "comma_operator.csv",
+    "conditional_operator.c": "conditional_operator.csv",
+    "implicit_predicate.c": "implicit_predicate.csv",
+    # "invalid_operator_precedence.c": "invalid_operator_precedence.csv",
+    "logic_as_controlflow.c": "logic_as_controlflow.csv",
+    # "macro_operator_precedence.c": "macro_operator_precedence.csv",
+    # "macro_test.c": "macro_test.csv",
+    "omitted_curly_braces.c": "omitted_curly_braces.csv",
+    # "operator_precedence.c": "operator_precedence.csv",
+    "post_incdec.c": "post_incdec.csv",
+    "pre_incdec.c": "pre_incdec.csv",
+    "repurposed_variable.c": "repurposed_variable.csv",
+    "reversed_subscripts.c": "reversed_subscripts.csv",
+    "type_conversion.c": "type_conversion.csv",
+}
+
+patch_path = Path.cwd() / "cocci"
```

Comment:
Using `cwd` should be avoided wherever possible. Again, you could use `__file__`